src/server/lib/validation-error.js

```diff
@@ -1,6 +1,6 @@
 
 class ValidationError extends Error {
-  constructor (message, { severity = 1, tab = null, row = null, col = null } = {}) {
+  constructor (message, { severity = 'err', tab = null, row = null, col = null } = {}) {
```

Comment:
Nit: I like using descriptive strings over numbers, but I also have a pet peeve for unnecessary abbreviations.  Could we just call it `'error'`?

Reply:
i'm mirroring what i did in https://github.com/usdigitalresponse/arpa-reporter/blob/3a3851f2f31b5580f1b86771424d6ecd111c61c6/src/components/AlertBox.vue#L20